tests/Unit/session_series-2.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Is this file actually used?

Reply:
Unless you plan on using it very soon I would remove it. At the very least change the filename and provide some sort of comment so you or anyone else knows what it is in a few months time.